### PR TITLE
Fix virtual keyboard over content

### PR DESCRIPTION
**Pull request by domi4484**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for da7eaca3da2af88d9f5142e6d3f54e05cd64ae97: [**arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-arm-neon.apk)
- [x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-x64.apk)
- [x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-x86.apk)
- [**all access arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-arm64.apk)
- [all access arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-arm-neon.apk)
- [all access x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-x64.apk)
- [all access x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5522-0c6fd898ce100e8701cec7d5f89aae8f12ae8ec0-android-x86.apk)
</details>

**Comment by domi4484** [comment]
@nirvn this doesn't help for me. Tried with google and samsung keyboard

**Comment by nirvn** [comment]
Qt 6.7 regression: https://bugreports.qt.io/browse/QTBUG-125323?jql=project%20%3D%20QTBUG%20AND%20text%20~%20%22Android%20keyboard%22%20ORDER%20BY%20created%20DESC%2C%20creator%20DESC

**Comment by nirvn** [comment]
That's a real bummer. Leaves us with the option of indefinitely delaying the next release until a fix is available or adding a bunch of overlays to go back to qt 6.6.3.

**Comment by m-kuhn** [comment]
> That's a real bummer. Leaves us with the option of indefinitely delaying the next release until a fix is available or adding a bunch of overlays to go back to qt 6.6.3.

Since the Google Play Store requires API level 34 for new apps, we are forced to switch from Qt 6.6 (API level 33) to Qt 6.7 (API level 34).

Comnent from the linked issue, it'snot honna help for long

File: platform/android/AndroidManifest.xml.in [hunk 1]
```diff
@@ -52,7 +52,8 @@
       android:launchMode="singleTask"
       android:configChanges="orientation|uiMode|screenLayout|screenSize|smallestScreenSize|locale|fontScale|keyboard|keyboardHidden|navigation"
       android:theme="@style/AppTheme"
-      android:exported="true">
+      android:exported="true"
+      android:windowSoftInputMode="adjustResize">
```
Comment (nirvn) [inline platform/android/AndroidManifest.xml.in:56]:
```suggestion
      android:windowSoftInputMode="adjustPan">
```

**Comment by nirvn** [comment]
Let's close this, I'm keeping track of the situation upstream.

**Pull request closed without merging**